mypy/test/testutil.py

```diff
@@ -0,0 +1,93 @@
+from typing import Iterator, List, Tuple
+import time
+import os
+import sys
+import tempfile
+from contextlib import contextmanager
+from threading import Thread
+from unittest import TestCase, main, skipUnless
```

Comment:
Wow, turns out `mypy.test`, `unittest`, *and* pytest are currently being used...ouch...